README.md

```diff
@@ -1,4 +1,6 @@
-# tgstation-server
```

Comment:
This line shouldn't have been deleted. Also there are now several unrelated changes present which need to be undone.

Reply:
All good, I think those new unrelated changes came from merging the dev branch into this one. Will revert those :)